Code/Framework/AzToolsFramework/Tests/ComponentModeTestDoubles.cpp

```diff
@@ -244,5 +254,10 @@ namespace AzToolsFramework
             return "OverrideMouseInteraction Edit Mode";
         }
 
+        AZ::Uuid OverrideMouseInteractionComponentMode::GetComponentModeType() const
+        {
+            return azrtti_typeid<OverrideMouseInteractionComponentMode>();
+        }
```

Comment:
update: As this is a breaking change, we should also make a note to include a deprecation notice (or provide a default implementation) so that clients who might have implemented their own Component Modes know how to upgrade.